graphql_compiler/tests/snapshot_tests/test_cost_estimation.py

```diff
@@ -1602,3 +1603,14 @@ def test_int_value_conversion_date(self):
                 schema_info, "Animal", "birthday", int_value
             )
             self.assertEqual(date_value, recovered_date)
+
+    def test_swap_uuid_prefix_and_suffix(self):
+        # Test with a uuid with lots of different characters
+        uuid_string = "01234567-89ab-cdef-0123-456789abcdef"
+        flipped_uuid = swap_uuid_prefix_and_suffix(uuid_string)
+        self.assertEqual("456789ab-cdef-cdef-0123-0123456789ab", flipped_uuid)
+
+        # Test with a different uuid in case repeating characters hid bugs
```

Comment:
I don't understand what you mean here.

Reply:
I was explaining why I chose that particular input, but on second thought I see that my way of thinking about it is more confusing than just not saying anything.